src/code42cli/cmds/devices.py

```diff
@@ -403,15 +404,21 @@ def _get_all_active_hold_memberships(sdk):
 def _get_device_dataframe(
     sdk, columns, active=None, org_uid=None, include_backup_usage=False
 ):
-    devices_generator = sdk.devices.get_all(
-        active=active, include_backup_usage=include_backup_usage, org_uid=org_uid
-    )
-    devices_list = []
-    if include_backup_usage:
-        columns.append("backupUsage")
-    for page in devices_generator:
-        devices_list.extend(page["computers"])
-    return DataFrame.from_records(devices_list, columns=columns)
+    try:
+        devices_generator = sdk.devices.get_all(
+            active=active, include_backup_usage=include_backup_usage, org_uid=org_uid
+        )
+        devices_list = []
+        if include_backup_usage:
+            columns.append("backupUsage")
+        for page in devices_generator:
+            devices_list.extend(page["computers"])
+        return DataFrame.from_records(devices_list, columns=columns)
+    except Py42BadRequestError as err:
```

Comment:
Ideally py42 would raise a Py42OrgNotFoundError here rather than Py42BadRequestError. Should we still go ahead with this fix for the device commands despite not having the preferred error abstraction available from py42?